setup.py

```diff
@@ -1,10 +1,11 @@
 from setuptools import setup
 
-VERSION = "1.5.2"
+VERSION = "1.6.0"
 
 install_requires = [
     "pygelf>=0.4.1",
-    "JSON-log-formatter==0.2.0"
+    "JSON-log-formatter==0.2.0",
+    "ddtrace==0.22.0" # This is the minimum version allowed as trace helpers weren't added until 0.22.0
```

Comment:
We should not be pinning versions in libraries, only in apps. You should use `>=` like I did with `pygelf`.

Reply:
Should I update `JSON-log-formatter` as well?

Reply:
Yes